NppJSONViewer/utils.h

```diff
@@ -1,5 +1,5 @@
 #pragma once
 #include <sstream>
 
-#define SSTR( x ) static_cast< std::ostringstream & >( \
-        ( std::ostringstream() << std::dec << x ) ).str()
+template<typename T>
```

Comment:
@kapilratnani while merging (Merge branch 'master' into src_MakeResourceAlignedWithVS) PR #45. This change was missed. Therefore, VS will report an error as project is now set to warning level 4 which will treat warning as error.

So, I have included the same change again in #46